Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml

```diff
@@ -0,0 +1,421 @@
+name: SecurityScorecard
+display: SecurityScorecard
+category: Vulnerability Management
+description: Provides scorecards for domains.
+commonfields:
+  id: SecurityScorecard
+  version: -1
+configuration:
+- name: username
+  display: Username/Email
+  additionalinfo: The SecurityScorecard username/email.
+  required: true
+  type: 9
+  displaypassword: API Token
+# Fetch Incident Parameters
+- name: isFetch
+  display: Fetch incidents
+  required: false
+  type: 8
+- name: incidentFetchInterval
+  display: Incidents Fetch Interval. SecurityScorecard is updated on a daily basis
+    therefore there's no need to modify this value.
+  defaultvalue: "1440"
+  required: false
+  type: 19
```

Comment:
remove this.
instead in the integration description mention that SecurityScorecard is updated on a daily basis